entry.tsx

```diff
@@ -7,7 +7,7 @@ import ReactDOM from 'react-dom';
 import {logError} from 'mattermost-redux/actions/errors';
 
 // Import our styles
-import 'sass/styles.scss';
+import './sass/styles.scss';
```

Comment:
why did this had to change though?

Reply:
This is the proper import path (since those files are local and not imported from the `sass` dependency), so I was getting during bundling the error:

```
[webapp] ERROR in ./entry.tsx 16:0-26
[webapp] Module not found: Error: Package path ./styles.scss is not exported from package /Users/pantelis/code/mattermost-webapp/node_modules/sass (see exports field in /Users/pantelis/code/mattermost-webapp/node_modules/sass/package.json)
[webapp] Did you mean './sass/styles.scss'?
```

Reply:
we didn't had that problem before, did we?

Reply:
Nope, the error happens due to the upgrade of the babel sass-loader.

Reply:
ahh i see